server/app/views/applicant/ProgramCardViewRenderer.java

```diff
@@ -236,11 +239,20 @@ private static Optional<ImgTag> createProgramImage(
     // TODO(#5676): Can we detect if the image URL is invalid and then not show it?
     // TODO(#5676): Include a placeholder while the image is loading.
 
+    LocalizedStrings altText;
```

Comment:
**suggestion (non-blocking):** There's a lot of characters here. How about pulling these out into a helper function to increase understanding?

`LocalizedStrings altText = getAltText(program);`

... and then an interested party can read that method if they are interested in the details

Reply:
Or even have the function return the `String` value needed by `.withAlt()` below, in which case you could inline the function call.

Reply:
Great suggestion,  done.